Week3/BusinessCards/app/src/main/java/com/example/businesscards/MainActivity.kt

```diff
@@ -2,10 +2,74 @@ package com.example.businesscards
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AlertDialog
+
 
 class MainActivity : AppCompatActivity() {
+    private val reminders = listOf("V = I x R", "I = V / R", "R = V / I", "P = V x I")
+
+    internal lateinit var currentReminder: String
+    private lateinit var changeReminderButton: Button
+    private lateinit var reminderTextView: TextView
```

Comment:
Please use Kotlin synthetics as we've described in the chat, or the ViewBinding approach.

`findViewById()` and lateinit vars are deprecated/very volatile, and they shouldn't be used!